src/caller/src/main.rs

```diff
@@ -277,6 +277,14 @@ fn get_project_from_ghidra(file_path: &Path, binary: &[u8]) -> (Project, Vec<Log
             }
         };
 
+        if !String::from_utf8(output.stdout.clone())
+            .unwrap()
```

Comment:
Can you change this unwrap to something like `unwrap_or_else(..)`, where you can tell Rust to `std::process::exit(101)` in the error case?